includes/api/class-edd-api.php

```diff
@@ -1581,6 +1581,7 @@ public function get_discounts( $discount = null ) {
 				$discount_list['discounts'][$count]['product_requirements']  = edd_get_discount_product_reqs( $discount->ID );
 				$discount_list['discounts'][$count]['requirement_condition'] = edd_get_discount_product_condition( $discount->ID );
 				$discount_list['discounts'][$count]['global_discount']       = edd_is_discount_not_global( $discount->ID );
+				$discount_list['discounts'][$count]['excluded_products']  	= edd_get_discount_excluded_products( $discount->ID );
```

Comment:
```suggestion
				$discount_list['discounts'][$count]['excluded_products']     = edd_get_discount_excluded_products( $discount->ID );
```
(Just adjusting mid-line alignment to use spaces.)